src/pages/veelgestelde-vragen.tsx

```diff
@@ -0,0 +1,108 @@
+import fs from 'fs';
+import Head from 'next/head';
+import path from 'path';
+import { Collapsable } from '~/components/collapsable';
+import { FCWithLayout, getLayoutWithMetadata } from '~/components/layout';
+import { MaxWidth } from '~/components/maxWidth';
+import siteText, { TALLLanguages } from '~/locale/index';
+import { MDToHTMLString } from '~/utils/MDToHTMLString';
+import { ensureUniqueSkipLinkIds, getSkipLinkId } from '~/utils/skipLinks';
+import styles from './over.module.scss';
+
+interface IVraagEnAntwoord {
+  vraag: string;
+  antwoord: string;
+  id?: string;
+}
+
+interface StaticProps {
+  props: {
+    text: TALLLanguages;
+    lastGenerated: string;
+  };
+}
+
+export async function getStaticProps(): Promise<StaticProps> {
+  const text = require('../locale/index').default;
+  const serializedContent = text.over_veelgestelde_vragen.vragen.map(function (
+    item: IVraagEnAntwoord
+  ) {
+    return {
+      ...item,
+      id: getSkipLinkId(item.vraag),
+      antwoord: MDToHTMLString(item.antwoord),
+    };
+  });
+
+  ensureUniqueSkipLinkIds(serializedContent);
```

Comment:
So this function mutating its arguments? If possible without much hassle I would much prefer it returns the new value instead of mutating, so it becomes something like:

```suggestion
 const processedContent = ensureUniqueSkipLinkIds(serializedContent);
```

Reply:
This re-applies this method. I'd be up for improving that method, but as a separate PR.